draft-chen-cfrg-vdaf-pine.md

```diff
@@ -323,6 +323,21 @@ The measurement encoding is done in two stages:
   Aggregators derive on their own, allow the Aggregators to run wraparound
   checks on their own.
 
+### Encoding Range-Checked Results {#encode-range-check}
```

Comment:
This basically says the same thing as the "Range Check" section below. What is your goal exactly?

Reply:
This section is more about Client encoding the range check. While the {{valid-range-check}} talks about what the Aggregator does to verify the range check.

Reply:
I cut some texts in {{valid-range-check}}.